services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -233,6 +234,7 @@ async def _start_dynamic_service(
         service_uuid=f"{node_uuid}",
         request_dns=extract_dns_without_default_port(request.url),
         request_scheme=request.headers.get("X-Forwarded-Proto", request.url.scheme),
```

Comment:
You again hardcode one header as a str, but another one you fetch as a (capital letter) config string